docs/semgrep-ci/network-broker.md

```diff
@@ -125,6 +125,34 @@ You can check the logs for Semgrep Network Broker by running:
 
 <pre class="language-console"><code>sudo docker logs <span className="placeholder">CONTAINER_ID</span></code></pre>
 
+### Adjusting log verbosity
+
+The Semgrep Network broker can log details of the proxied requests and responses for troubleshooting. Normally, these should only be collected when working to identify an issue. If requests and responses are large, they can occupy significant memory in the tunnel.
+
+In your broker configuration, that looks like:
```

Comment:
Is this an example we want people to copy?

Reply:
yep, they can safely copy this example. the one below is also syntactically fine although wouldn't be very useful

Reply:
Got it. It sounded a little hypothetical (or rather I wasn't sure if I would use it) - maybe we can be more explicit that the reader can use the snippet? Minor suggestion only.

Reply:
How would you phrase that? I don't have great idea of how to make it more clear, especially given it follows a caution only to use it for debugging. :)